backend/FacilityBooking/FacilitiesAPI.py

```diff
@@ -268,8 +268,15 @@ def add_booking():
             {"_id": 0}
         ).count()
 
+        conflictadd = []
+        conflictadd.append( {
```

Comment:
This does not work. The condition below that you copied is the query condition to the DB, not the data we want to return. Basic testing of the app would have given you 
{
    "err": "append() takes exactly one argument (2 given)",
    "status": "failed"
}.

We want to return the array of conflicted bookings from the DB.